hinted_handoff_test.py

```diff
@@ -0,0 +1,60 @@
+import time
+from avocado import main
+from sdcm.tester import ClusterTester
+
+
+class HintedHandoffTest(ClusterTester):
+    """
+       :avocado: enable
+    """
+    def __init__(self, *args, **kwargs):
+        super(HintedHandoffTest, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+        super(HintedHandoffTest, self).setUp()
+        self.start_time = time.time()
+
+    def test_stop_nodes_under_stress(self):
+        """
+            3 nodes cluster, RF=3.
+            While Write n=X with CL=1.
+            Stop node1
+            Wait some time
+            Stop node2
+            Wait till write finish
+            Start all stopped nodes (1 & 2).
+            Wait for hints to be sent.
+            Stop node3.
+            Read all data n=X with CL=ONE.
+        """
+        assert len(self.db_cluster.nodes) == 3, "The test requires 3 DB nodes!"
+        node1 = self.db_cluster.nodes[0]
+        node2 = self.db_cluster.nodes[1]
+        node3 = self.db_cluster.nodes[2]
+        stress_write_cmd = self.params.get('stress_write_cmd')
```

Comment:
A thought, the test looks like tightly coupled with its yaml file, so maybe we shouldn't store the values in the Yaml and have them hard coded here?